src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1055,6 +1167,18 @@ void SetUpZoneSizingArrays(EnergyPlusData &state)
             state.dataSize->FinalZoneSizing(CtrlZoneNum).ZoneSecondaryRecirculation =
                 state.dataSize->ZoneSizingInput(ZoneSizNum).ZoneSecondaryRecirculation;
             state.dataSize->FinalZoneSizing(CtrlZoneNum).ZoneVentilationEff = state.dataSize->ZoneSizingInput(ZoneSizNum).ZoneVentilationEff;
+            finalZoneSizing.zoneSizingMethod = zoneSizingInput.zoneSizingMethod;
```

Comment:
Using references only with the new code initially is greatly appreciated. I would suggest that any further mass referencing wait until after this is merged.